Subject: The Encode::TW::Unisys::*.pm modules had no documentation at all.
 
 * lib/Encode/TW/Unisys/SOSI1.pm: Enhanced documentation.
 * lib/Encode/TW/Unisys/SOSI2.pm: Enhanced documentation.

Bug: http://rt.cpan.org/Public/Bug/Display.html?id=60137
Author: Nicholas Bamber <nicholas@periapt.co.uk>
Last-Update: 2010-08-06
--- a/lib/Encode/TW/Unisys/SOSI1.pm
+++ b/lib/Encode/TW/Unisys/SOSI1.pm
@@ -40,6 +40,10 @@
 
 =head1 NAME
 
-Encode::Unisys::SOSI1
+Encode::Unisys::SOSI1 - Internal module for unisys_sosi1 characters.
+
+=head1 DESCRIPTION
+
+See L<Encode::HanExtra> for more information.
 
 =cut
--- a/lib/Encode/TW/Unisys/SOSI2.pm
+++ b/lib/Encode/TW/Unisys/SOSI2.pm
@@ -40,6 +40,9 @@
 
 =head1 NAME
 
-Encode::Unisys::SOSI2
+Encode::Unisys::SOSI2 - Internal module for unisys_sosi2 characters
 
+=head1 DESCRIPTION
+
+See L<Encode::HanExtra> for more information.
 =cut
